src/components/icons/mask.vue

```diff
@@ -0,0 +1,12 @@
+            fill="#4A577B"
+        />
+    </svg>
+</template>
+
+<script lang="ts">
+export default {};
```

Comment:
😩 I'ts kind of sad that we need to do that

Reply:
It is sad...